src/main/java/com/aws/greengrass/util/platforms/windows/WindowsExec.java

```diff
@@ -133,9 +137,6 @@ public synchronized void close() throws IOException {
 
     @SuppressFBWarnings("SWL_SLEEP_WITH_LOCK_HELD")
     private void stopGracefully() {
-        if (!process.isAlive()) {
```

Comment:
Why remove this?

Reply:
We just checked isAlive right before this. On L123